HLTrigger/HLTcore/interface/TriggerExpressionData.h

```diff
@@ -224,7 +223,7 @@ class Data {
 
   // l1 values and status
   const L1GlobalTriggerReadoutRecord  * m_l1tResults;
-  const L1GtTriggerMenu               * m_l1tMenu;
+  std::unique_ptr<L1GtTriggerMenu>      m_l1tMenu;
```

Comment:
Ah, I see it can't because of the call to ```buildGtConditionMap```. Would it be possible to change ```L1GtTriggerMenu``` to have a constructor which calls that function?